packages/lib/RotatingLogs.test.ts

```diff
@@ -35,11 +35,28 @@ describe('RotatingLogs', () => {
 			await rotatingLogs.cleanActiveLogFile();
 			await msleep(1);
 			await rotatingLogs.deleteNonActiveLogFiles();
-			const files = await readdir(dir);
+			const files: string[] = await readdir(dir);
```

Comment:
Please don't set the type when it can be inferred by the compiler. I think I mentioned this before? You do that a lot and this can actually disable typechecking if the function you're calling changes its return type later on.

Reply:
Oh, it explains why it wasn't typed. Sorry, I thought this rule just applied to type `any`. I believe I understood it better now. Thanks and sorry for this kind of mistake.